pkg/cmd/pr/merge/merge.go

```diff
@@ -192,9 +194,12 @@ func mergeRun(opts *MergeOptions) error {
 	}
 
 	isPRAlreadyMerged := pr.State == "MERGED"
-	if blocked := blockedReason(pr.MergeStateStatus); !opts.AutoMergeEnable && !isPRAlreadyMerged && blocked != "" {
-		fmt.Fprintf(opts.IO.ErrOut, "%s Pull request #%d is not mergeable: %s.\n", cs.FailureIcon(), pr.Number, blocked)
+	if !opts.AutoMergeEnable && !isPRAlreadyMerged && isBlocked(pr.MergeStateStatus, opts.Admin) {
```

Comment:
Instead of a new function `isBlocked` which mostly duplicates `blockedReason`, could `blockedReason` simply accept an extra `Admin` argument and adjust logic accordingly?

Reply:
Agree on that, thought about a double return as well after i commited it.
Though the second return is not even necessary if  BEHIND is also blocking the merge (see above).

Reply:
Changed `blockedReason` accordingly but i am not to sure about the additional `bool` return (keeps it concise though).
Open for ideas.